test/funnels/Depositor.t.sol

```diff
@@ -156,6 +155,26 @@ contract DepositorTest is DssTest, TestUtils {
         assertEq(NftLike(UNIV3_POS_MGR).balanceOf(address(buffer)), 1);
     }
 
+    function testDepositAfterHop() public {
```

Comment:
I think we should probably discard the changes in this file so we can only focus on complete the Swapper part as the Depositor will probably change anyway.

Reply:
Done.
The changes are now only part of https://github.com/makerdao/dss-allocator/pull/15.